metadata.yaml

```diff
@@ -7,6 +7,12 @@ summary: |
   Kubernetes charm for MySQL
 series:
   - kubernetes
+  - mysql
+  - db
```

Comment:
I think series means what platform the charm runs on. Kubernetes, Ubuntu, CentOS, etc.

Reply:
Yes that is correct. I think mysql and db should not be in series.